packages/itwin/grouping-mapping-widget/src/widget/components/GroupPropertyAction.scss

```diff
@@ -4,26 +4,26 @@
 *--------------------------------------------------------------------------------------------*/
 @import '~@itwin/itwinui-css/scss/variables';
 
-.group-property-action-container {
+.gmw-group-property-action-container {
   display: flex;
   flex-direction: column;
   overflow-x: auto;
   height: 100%;
 
-  .property-options {
+  .gmw-property-options {
     display: inherit;
     flex-direction: inherit;
     gap: $iui-baseline;
     margin-bottom: $iui-baseline * 4;
   }
 
-  .property-selection-container {
+  .gmw-property-selection-container {
     display: flex;
     flex-direction: column;
     gap: $iui-baseline;
 
-    .property-select-item {
-      .selection-and-reorder {
+    .gmw-property-select-item {
+      .gmw-selection-and-reorder {
         display: flex;
         gap: $iui-s;
       }
```

Comment:
There is a style for `.iui-text-leading` selector below in this file which might cause conflicts in other widgets or components. Why is it needed? If it is really needed, you should think about how to reduce the scope of that selector.

Reply:
It is not needed. Remove please.

Reply:
Done